src/app/constant.js

```diff
@@ -10,6 +10,14 @@
 // import { version } from '../../package.json';
 import version from '../version';
 
+/**
+ * @since v3.7.0
+ */
+
+import eventbus from "./eventbus";
+
+export const APP_EVENTBUS = eventbus;
+
```

Comment:
Re-evaluate the implications

Reply:
@Raruto need it to use a central event bus application used by plugins

Reply:
@volterra79 Let me know if doing like so could be okay too 👉 https://github.com/g3w-suite/g3w-client-plugin-editing/pull/52/commits/178a3fd9cae1fa77d44050feda3684aa7fbf34af + https://github.com/g3w-suite/g3w-client/pull/483/commits/6818ac30ed2ee43b89a7088a548a05f7cdb86526 

Unless strictly necessary, I would prefer avoid exposing new global stuff.